code/game/objects/obj_defense.dm

```diff
@@ -95,8 +95,14 @@
 
 /obj/attack_slime(mob/living/simple_animal/slime/user, list/modifiers)
 	if(!user.is_adult)
+		if(user.client)
+			log_combat(user, src, "attempted to attack")
```

Comment:
why does this need to be logged

Reply:
To predict shit happening, but i can remove it